app/api/dao/user.py

```diff
@@ -157,11 +157,14 @@ def list_users(user_id, is_verified=None):
             list_of_users = [user.json() for user in users_list]
 
         for user in list_of_users:
```

Comment:
Want to make it even better? Check `map` from lamda expressions.

Reply:
Actually I do not have access to my pc right now, trying it on mobile and giving errors. I would check if it works here else I would be updating it soon.

Reply:
> Want to make it even better? Check `map` from lamda expressions.

Excellent suggestion! Works great, thank you for this:)

Reply:
👍 fantastic!